package.json

```diff
@@ -21,16 +21,17 @@
     "test:e2e": "jest --config ./test/jest-e2e.json --runInBand",
     "test:e2e:cov": "jest --coverage --config ./test/jest-e2e.json --runInBand",
     "test:all": "jest --config ./test/jest-all.json --runInBand",
-    "test:all:cov": "jest --coverage --config ./test/jest-all.json --runInBand",
-    "postinstall": "husky install"
+    "test:all:cov": "jest --coverage --config ./test/jest-all.json --runInBand"
   },
   "dependencies": {
+    "@nestjs/cli": "^10.0.5",
```

Comment:
This is included as a production dependency since `nest build` is used when building the image. This could be changed by using `tsc` directly in the future, but I left this task out of the scope of this PR.